### PR TITLE
Briefing room fixes

### PR DESCRIPTION
**Pull request by AaronGullickson**
This PR addresses several graphical issues in the Briefing Room tab. The main fix is for #1768 where the TO&E reports for a scenario were being cut off. However, I was also annoyed about how the various text areas were leaving a bunch of white space, so I made some changes to the grid bag layout to fix that.

Here is a shot of what the it looks like before:
Screen Shot 2020-05-27 at 2 06 21 PM

Here is a shot of what it looks like after:
Screen Shot 2020-05-27 at 2 05 45 PM

I did have to unsnarl a bit of code to get it all working. Most importantly I passed the scenarioTable object into the MissionViewPanel where it is displayed so that there is only one scroll bar on the left. In order to do that without duplicating a bunch of code, I removed the separate ContractViewPanel and AtBContractViewPanel classes and consolidated them with MissionViewPanel. The only area where they differed was the `fillStats` function so this was relatively straightforward.

### CODE REVIEW

File: MekHQ/src/mekhq/gui/view/MissionViewPanel.java [hunk 1]
```diff
@@ -9,15 +9,17 @@
 import java.awt.*;
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/gui/view/MissionViewPanel.java:9]: Ended up mixed tabs and spaces.

Comment (AaronGullickson) [reply MekHQ/src/mekhq/gui/view/MissionViewPanel.java:9]: Just switched over to intelliJ and probably need to futz with some settings

Comment (sixlettervariables) [reply MekHQ/src/mekhq/gui/view/MissionViewPanel.java:9]: I need to get us an editorconfig so they'll all just work.

Comment (AaronGullickson) [reply MekHQ/src/mekhq/gui/view/MissionViewPanel.java:9]: Tabs should all be gone now.

**Pull request merged**